pkg/cloudprovider/vmss_test.go

```diff
@@ -22,3 +22,28 @@ func TestSetUserIdentities(t *testing.T) {
 	update = testIdentityInfo.SetUserIdentities(map[string]bool{"id3": true, "id4": true, "id1": false})
 	assert.True(t, update)
 }
+
+func TestRemoveUserIdentityVMSS(t *testing.T) {
+	testIdentityInfo := &vmssIdentityInfo{
+		info: &compute.VirtualMachineScaleSetIdentity{
+			UserAssignedIdentities: map[string]*compute.VirtualMachineScaleSetIdentityUserAssignedIdentitiesValue{
+				"ID1": nil,
```

Comment:
Lets change the value to `&compute.VirtualMachineScaleSetIdentityUserAssignedIdentitiesValue`

Reply:
Changed to `{}` since gofmt is complaining about `redundant type from array, slice, or map composite literalsimplifycompositelit` when I used `&compute.VirtualMachineScaleSetIdentityUserAssignedIdentitiesValue`.